server/startup/accounts.js

```diff
@@ -120,9 +120,14 @@ export default function () {
         } else {
           group = Collections.Groups.findOne({ slug: "customer", shopId });
         }
+        if (group) {
+          roles[shopId] = group.permissions;
+          additionals.groups = [group._id];
+        }
         // if no group or customer permissions retrieved from DB, use the default Reaction customer set
-        roles[shopId] = group.permissions || Reaction.defaultCustomerRoles;
```

Comment:
new test was failing on this line, where `group` was null.
While I don't understand how similar tests were passing, an extra null check can't hurt.